bayesian/utils.py

```diff
@@ -103,6 +103,20 @@ def server_create_component_bookkeeping(ecosystem, name, version, user_profile):
     return server_run_flow('componentApiFlow', args)
 
 
+def create_component_bookkeeping(ecosystem, packages_list, user_id, user_agent):
+    """Run the component analysis for given ecosystem+package+version."""
+    args = {
+        'api_name': 'component_analyses_post',
+        'external_request_id': uuid.uuid4().hex,
```

Comment:
My suggestion will be to generate this on the client side and use this for all the POST requests for a single stack. With this we will be able to figure out the stack.

Are we not populating the manifest_hash

Reply:
For the manifest_hash I will add codeto fetch it from the user.